Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml

```diff
@@ -971,6 +971,13 @@
   sprites:
     - sprite: Mobs/Customization/human_hair.rsi
       state: fringetail
+- type: marking
+  id: HumanHairGatheredlow
+  bodyPart: Hair
+  markingCategory: Hair
+  sprites:
+    - sprite: Mobs/Customization/human_hair.rsi
```

Comment:
change this to _impstation/mobs/customization/human_hair.rsi and it wont error when you migrate the rest of the file structure

Reply:
feel free to use pr #548 for reference on how to add cosmetics